vars/commonPipelineEnvironment.groovy

```diff
@@ -19,6 +19,9 @@ class commonPipelineEnvironment implements Serializable {
     //Stores the current buildResult
     String buildResult = 'SUCCESS'
 
+    //Stores a feature toggle for defaulting to technical names in stages
+    boolean useTechnicalStageNames
```

Comment:
We previously used env var for feature toggles.

Reply:
The Cloud SDK Pipeline would always enable this. It is not something we want the user to configure or even be able to change. Is there a way to do this via env var without exposing it to the user in the pipeline?

Reply:
I have now moved everything into a new singleton class `StageNameProvider`. The field of this class can be accessed from steps. Everything related is now in this one class and the CPE remains unchanged.